CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [v1.4.1](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/v1.4.1) (2024-12-20)
+
+[Full Changelog](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/compare/v1.4.0...v1.4.1)
+
+**Implemented enhancements:**
+- Allows setting the provider host and authorization_token via CAPELLA prefixed environment variables [\#239](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/239) ([cdsre](https://github.com/cdsre))
```

Comment:
```suggestion
- Allows the provider `host` and and `authentication_token` to be set through environment variables prefixed with `CAPELLA_HOST` and `CAPELLA_AUTH_TOKEN`. [\#239](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/239) ([cdsre](https://github.com/cdsre))
```

Reply:
Similar to what @simon-dew suggested and after going through the PR linked here, I set up the variables as written in the PR.

- `authorization_token` is actually `authentication_token` in the PR.
- The CAPELLA environment variables in the PR are written as `CAPELLA_HOST` and `CAPELLA_AUTH_TOKEN`. 

I just want to confirm that these variable are still written as this?